plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -111,6 +113,24 @@
 public class IcebergMetadata
         implements ConnectorMetadata
 {
+    private static final Logger log = Logger.get(IcebergMetadata.class);
+    private static final String ICEBERG_MATERIALIZED_VIEW_COMMENT = "Presto Materialized View";
+    public static final String DEPENDS_ON_TABLES = "dependsOnTables";
+
+    // Be compatible with views defined by the Hive connector, which can be useful under certain conditions.
+    private static final String TRINO_CREATED_BY = HiveMetadata.TRINO_CREATED_BY;
+    private static final String TRINO_CREATED_BY_VALUE = "Trino Iceberg connector";
+    private static final String PRESTO_VIEW_COMMENT = HiveMetadata.PRESTO_VIEW_COMMENT;
+    private static final String PRESTO_VERSION_NAME = HiveMetadata.PRESTO_VERSION_NAME;
+    private static final String PRESTO_QUERY_ID_NAME = HiveMetadata.PRESTO_QUERY_ID_NAME;
+    private static final String PRESTO_VIEW_EXPANDED_TEXT_MARKER = HiveMetadata.PRESTO_VIEW_EXPANDED_TEXT_MARKER;
+
+    private static final String PARQUET_BLOOM_FILTER_COLUMNS_KEY = "parquet.bloom.filter.columns";
```

Comment:
Changes for iceberg should be in separate commit. Ideally we should try to land the change in hive connector first and then attempt iceberg in follow up PR.